out/extension.js

```diff
@@ -227,10 +228,14 @@ function activate(context) {
 
     // Notebook
     context.subscriptions.push(vscode.workspace.registerNotebookSerializer('splunk-notebook', new notebookSerializers.SplunkNotebookSerializer(), {transientCellMetadata: {inputCollapsed: true, outputCollapsed: true}, transientOutputs: false}));
-	let controller = new notebookControllers.SplunkController()
-    context.subscriptions.push(controller)
-    context.subscriptions.push(vscode.notebooks.registerNotebookCellStatusBarItemProvider('splunk-notebook', new notebookProviders.CellResultCountStatusBarProvider(splunkOutputChannel)));
-    notebookCommands.registerNotebookCommands(controller, splunkOutputChannel, context)
+	context.subscriptions.push(vscode.workspace.registerNotebookSerializer('spl2-notebook', new notebookSerializers.SplunkNotebookSerializer(), {transientCellMetadata: {inputCollapsed: true, outputCollapsed: true}, transientOutputs: false}));
+    const controller = new notebookController.SplunkController();
+    context.subscriptions.push(controller);
+    const spl2Controller = new notebookSpl2Controller.Spl2Controller();
+    context.subscriptions.push(spl2Controller);
+    context.subscriptions.push(vscode.notebooks.registerNotebookCellStatusBarItemProvider('splunk-notebook', new notebookProvider.CellResultCountStatusBarProvider(splunkOutputChannel)));
+    context.subscriptions.push(vscode.notebooks.registerNotebookCellStatusBarItemProvider('spl2-notebook', new notebookProvider.CellResultCountStatusBarProvider(splunkOutputChannel)));
```

Comment:
maybe?
```suggestion
    context.subscriptions.push(vscode.notebooks.registerNotebookCellStatusBarItemProvider('spl2-notebook', new notebookSpl2Controller.CellResultCountStatusBarProvider(splunkOutputChannel)));
```

Reply:
I tweaked this slightly but the same notebook provider is used for both actually so this isn't needed